neo4j/example/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-data-neo4j-example</artifactId>
+    <name>Spring Data Neo4j - Example</name>
+    <properties>
+        <spring-data-neo4j.version>4.0.0.BUILD-SNAPSHOT</spring-data-neo4j.version>
+        <neo4j-ogm.version>1.1.2-SNAPSHOT</neo4j-ogm.version>
+        <neo4j.version>2.2.4</neo4j.version>
+    </properties>
+
+    <parent>
+        <groupId>org.springframework.data.examples</groupId>
+        <artifactId>spring-data-neo4j-examples</artifactId>
+        <version>1.0.0.BUILD-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-neo4j</artifactId>
+            <version>${spring-data-neo4j.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
What's the reason we need these test JARs in the classpath? I would've expected to only see the Spring Data Neo4j dependency. Anything I am missing?